js/app/widgets/dynamicLogo.js

```diff
@@ -135,6 +152,10 @@ const DynamicLogo = new Knowledge.Class({
     },
 
     _update_custom_style: function () {
+        this._max_width = EosKnowledgePrivate.style_context_get_custom_int(this.get_style_context(), 'max-width');
+        this._max_height = EosKnowledgePrivate.style_context_get_custom_int(this.get_style_context(), 'max-height');
+        this.queue_resize();
```

Comment:
This queues a resize anytime the style is updated, even if max-width and max-height didn't change.